lib/api/include/irods/getHostForPut.h

```diff
@@ -11,4 +11,9 @@ extern "C"
 #endif
 int rcGetHostForPut( rcComm_t *conn, dataObjInp_t *dataObjInp, char **outHost );
 
+#ifdef __cplusplus
+extern "C"
+#endif
+int rc_get_rescinfo_for_put( rcComm_t *conn, dataObjInp_t *dataObjInp, char **outHost );
```

Comment:
Does this make the original API obsolete?

Reply:
possibly. should we get rid of `rxGetHostForPut` ?

Reply:
No. Let's keep them. Some clients may rely on them.